psx_scraper.py

```diff
@@ -37,12 +37,27 @@
 except FileNotFoundError:
     pass
 
+def test_pbp(file: bytes):
+    """function to test file for psx-pbp information"""
+    try:
+        # read the entire file
+        file.seek(0)
```

Comment:
This is a needlessly expensive operation, there's no need to read the whole file just for those two bytestrings.  I suggest fixing this so it jumps to the proper offset and stops reading once the string in question is found.

Reply:
Yes, I'm reading the entire file because the information seem to not be located in the same place from file to file, at least in the ones I've tested. I'm gonna take a second look later to see if I can actually detect a pattern to avoid reading the entire file.

Reply:
Gotcha, I was afraid of that.  Does the position change based on region or is it just random? 

Also, even if we can't predict position we can still stop reading once the bytestring is found.

Reply:
> 
> 
> Gotcha, I was afraid of that. Does the position change based on region or is it just random?

That's what I'm going to investigate, I just ran a test in my library without actually looking for patterns, but I'm pretty sure it could be a region thing.

> Also, even if we can't predict position we can still stop reading once the bytestring is found.

Yeah, didn't think about this solution, I'm also gonna test this solution and compare performance against reading the entire file, my fear is that testing byte by byte will actually make it slower than just reading the entire file.

Reply:
Reading full file: 0.026s
Reading one byte at a time: 6.924s

You're absolutely right, it's WAY slower to do this one byte at a time.  Dunno why I didn't think of that sooner.  I think your solution is acceptable as-is, and I'm going to change get_game_name to adopt that behavior as well.  Good catch!